person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permision: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permision = parent_permision
+  end
+
+  private
```

Comment:
- [ ] Every instance method after private becomes a private method. So please move the can_use_services method above the private keyword.